### ISSUE TITLE
Incorrect control type is announced for the edit "Choose to configure settings by azure subscription or provide them manually" 

### ISSUE THREAD

**Issue by prasadaps** [microsoft/vscode#126895]
[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.

GitHubTags:#A11y_VSCode;#A11ySev3;#A11yTCS;#A11yWCAG;#DesktopApp;#Visual Studio Code Client;#JAWS;#WCAG4.1.2;#DesktopApp;#Win10;#VisualStudioCode-Win32-Apr2021;

### Environment details:
Application Name: Azure Stream Analytics Tools for Visual Studio Code
Windows Version: Win10 
Visual studio code  version : -1.55.1(User Setup)
Screenreader NVDA(2020.4), JAWS(2021.2102.34ILM)

### Repro Steps:
1. Launch Visual Studio code application with installed Azure Stream Analytics Tools extension . 
2. Tab to "Explorer" section in activity bar. 
3. Press control +shift +P and edit bar will appear. 
4. Type "ASA:Create new project" and press enter to create project. 
5. New project will get created. 
6. Tab to Input item in Project Tree Section 
7. Right click on input and navigate to "ASA : Add Input" using arrow key and press enter. 
8. Select IoT hub from available option using enter key.
9. Start screenreader and check control type of "select from azure subscriptions" edit field.

### Actual Result:
Incorrect control type is announced for the edit "Choose to configure settings by azure subscription or provide them manually" as combo box.
Note this issue is observed in both screen reader nvda and jaws.

### Expected Result:
A control type announced for the edit "Choose to configure settings by azure subscription or provide them manually" should be edit field.

### User Impact:
The screen reader users may get confused with functionality of "Choose to configure settings by azure subscription or provide then manually" edit field.

### Attachment:
[A11y_AzureStreamAnalyticsToolsForVisualStudioCode_Input_ConfigureSettings.zip](https://github.com/microsoft/vscode-docs/files/6693549/A11y_AzureStreamAnalyticsToolsForVisualStudioCode_Input_ConfigureSettings.zip)

**Comment by roblourens** [comment]
I think combo box might actually be correct here. It doesn't look like a typical one, but that control is functionally equivalent to a combo box. @isidorn?

**Comment by isidorn** [comment]
Please file this against the Azure Stream Analytics Tools extension.
Then they can think about this issue and if the problem is on the vscode side they can translate this issue to VS Code language and create an issue in our repo themselves. Thanks
